test/integration_tests/abao/abao_test_hooks.js

```diff
@@ -48,6 +48,7 @@ hooks.skip("POST /download -> 200");
 hooks.skip("GET /download -> 200");
 hooks.skip("POST /upload/label -> 200");
 hooks.skip("POST /upload/uid -> 200");
+hooks.skip("POST /upload/uid -> 404");
```

Comment:
In order to cause a 404, we'll need to send metadata via a file form so I thought it lined up well enough.